cliquet/events.py

```diff
@@ -0,0 +1,22 @@
+from cliquet.utils import strip_uri_prefix, Enum
+
+
+ACTIONS = Enum(CREATE='create',
+               DELETE='delete',
+               READ='read',
+               UPDATE='update')
+
+
+class ResourceChanged(object):
+    """Triggered when a resource is changed.
+    """
+    def __init__(self, action, resource, request):
+        self.request = request
+        self.payload = {'timestamp': resource.timestamp(),
```

Comment:
what is "resource" ? please give me the name of the class I should have here. because below you are using resource.name and I can't find it in the code. timestamp is a function in collection